src/commands/setheaderrowcommand.js

```diff
@@ -171,7 +170,9 @@ function splitHorizontally( tableCell, headingRows, writer ) {
 		if ( columnIndex !== undefined && columnIndex === column && row === endRow ) {
 			const tableRow = table.getChild( row );
 
-			writer.insertElement( 'tableCell', attributes, Position.createFromParentAndOffset( tableRow, cellIndex ) );
+			const newCell = writer.createElement( 'tableCell', attributes );
+			writer.insert( newCell, Position.createFromParentAndOffset( tableRow, cellIndex ) );
+			writer.insertElement( 'paragraph', Position.createAt( newCell, 0 ) );
```

Comment:
The same as https://github.com/ckeditor/ckeditor5-table/pull/97/files#r206859198